django_project/feti/static/feti/css/landing-page.css

```diff
@@ -546,7 +546,7 @@
 .result-title img {
     width: 54px;
     height: auto;
-    margin-right: 20px;
+    margin: 10px 5px 5px 5px;
```

Comment:
hi @ann26 
you could use table for that
and make valign=center

try in here : https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_td_valign